runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/SkillCalculator.java

```diff
@@ -199,34 +213,45 @@ private void renderBonusOptions()
 	private void renderActionSlots()
 	{
 		// Wipe the list of references to the slot components.
+		Stream.concat(uiActionSlotBoxes.stream(), uiActionSlots.stream())
```

Comment:
What about having parent component and calling removeAll? this seems really suboptimal